apps/webapp/app/send/ibc-form.tsx

```diff
@@ -0,0 +1,81 @@
+import { useStore } from '../../state';
+import { ibcSelector } from '../../state/ibc';
+import { chains } from './constants';
+import { FilledImage, InputToken } from '../../shared';
+import { validateAmount } from '../../utils';
+import { Button, Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from 'ui';
+
+export const IbcForm = () => {
+  const { amount, asset, chain, validationErrors, setAmount, setAsset, setChain } =
+    useStore(ibcSelector);
+
+  return (
+    <form
+      className='flex flex-col gap-2'
+      onSubmit={e => {
+        e.preventDefault();
+      }}
+    >
+      <InputToken
+        label='Amount to send'
+        placeholder='Enter an amount'
+        className='mb-1'
+        asset={asset}
+        setAsset={setAsset}
+        value={amount}
```

Comment:
<img width="188" alt="Screenshot 2023-10-06 at 10 20 46 AM" src="https://github.com/penumbra-zone/web/assets/16624263/fdb50773-e002-4c9a-975a-a38a8b3fcf1b">

Is the font of the amount correct? 🤔

Reply:
I am waiting for the ui kit from Daria